config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-presubmit-dra-capz.yaml

```diff
@@ -72,6 +72,8 @@ presubmits:
           value: "Standard_D8s_v3"
         - name: TEST_WINDOWS
           value: "false"
+        - name: MONITORING_MACHINE_COUNT
+          value: "1" # One monitoring machine for prometheus server
         # Don't install Azure disk CSI driver as it's installed using a HelmChartProxy
```

Comment:
Should we preemptively update `CLUSTER_TEMPLATE` to the machine-pool one? Do we plan to run the presubmit outside of https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5784 before merging that?

Reply:
`CLUSTER_TEMPLATE="test/dev/cluster-template-custom-builds-machine-pool-load-dra.yaml"` if so

Reply:
> /hold depending on when we want to set CLUSTER_TEMPLATE for the presubmits.

We can follow up on this later.

/hold cancel